ocs_ci/utility/version.py

```diff
@@ -152,3 +153,115 @@ def compare_versions(expression):
         )
     v1, op, v2 = m.groups()
     return eval(f"get_semantic_version(v1, True){op}get_semantic_version(v2, True)")
+
+
+def get_dr_hub_operator_version(namespace="openshift"):
+    """
+    Get DR Hub Opertaor Version
+
+    Returns:
+        str: returns version string
+    """
+    # Importing here to avoid circular dependency
+    from ocs_ci.ocs.resources.csv import get_csvs_start_with_prefix
+
+    csv_list = get_csvs_start_with_prefix(
+        constants.ACM_ODR_HUB_OPERATOR_RESOURCE, namespace=namespace
+    )
+    for csv in csv_list:
+        if constants.ACM_ODR_HUB_OPERATOR_RESOURCE in csv["metadata"]["name"]:
+            # extract version string
+            return csv["spec"]["version"]
+
+
+def get_ocp_dr_cluster_operator_version(namespace="openshift"):
+    """
+    Get DR Hub Opertaor Version
+
+    Returns:
+        str: returns version string
+    """
+    # Importing here to avoid circular dependency
+    from ocs_ci.ocs.resources.csv import get_csvs_start_with_prefix
+
+    csv_list = get_csvs_start_with_prefix("odr-cluster-operator", namespace=namespace)
+    for csv in csv_list:
+        if "odr-cluster-operator" in csv["metadata"]["name"]:
+            # extract version string
+            return csv["spec"]["version"]
+
+
+def get_odf_multicluster_orchestrator_version(namespace=constants.ACM_HUB_NAMESPACE):
+    """
+    Get ODF Multicluster Orchestrator Version
+
+    Returns:
+        str: returns version string
+    """
+    # Importing here to avoid circular dependency
+    from ocs_ci.ocs.resources.csv import get_csvs_start_with_prefix
+
+    csv_list = get_csvs_start_with_prefix(
+        constants.ACM_ODF_MULTICLUSTER_ORCHESTRATOR_RESOURCE,
+        namespace=namespace,
+    )
+    for csv in csv_list:
+        if (
+            constants.ACM_ODF_MULTICLUSTER_ORCHESTRATOR_RESOURCE
+            in csv["metadata"]["name"]
+        ):
+            # extract version string
+            return csv["spec"]["version"]
+
+
+def get_ocp_gitops_operator_version(namespace="openshift"):
```

Comment:
is "openshift" namespace for DR ? don't make namespace as hardcode since we support multi namespace feature.

Reply:
done